test/grid/test_centroids.py

```diff
@@ -13,10 +13,13 @@
     _normalize_xyz,
 )
 
-current_path = Path(os.path.dirname(os.path.realpath(__file__)))
+# Import centralized paths
+import sys
+sys.path.append(str(Path(__file__).parent.parent))
```

Comment:
Are these two lines necessary?

Reply:
Yeah, I like that idea to have a cleaner path access with imports eliminated.